### ISSUE TITLE
2.1 HTML Illegal page dimensions if setting 1 dimension 

### ISSUE THREAD

**Issue by klabarge** [qzind/tray#439]
Using 2.1 (RC6), If only one dimension (`height` or `width`) is supplied, an `Illegal dimension` error is thrown with HTML printing only.

* Does not happen in 2.0
* Does not happen using PDF or Image printing

```js
var config = qz.configs.create("Printer Name", { size: {width: 4} });
var data = [{
   type: 'pixel',
   format: 'html',
   flavor: 'file',
   data: 'test.html'
}];
qz.print(config, data).catch(function(e) { console.error(e); });
```

```
[ERROR] 2019-04-09 19:38:34,927 @ qz.utils.PrintingUtilities:?
	Failed to print
java.lang.IllegalArgumentException: Illegal dimension
	at javafx.print.Paper.<init>(Paper.java:56)
	at javafx.print.Printer$1.createPaper(Printer.java:424)
	at com.sun.javafx.print.PrintHelper.createPaper(PrintHelper.java:58)
	at qz.printer.action.PrintHTML.print(Unknown Source)
	at qz.utils.PrintingUtilities.processPrintRequest(Unknown Source)
	at qz.ws.PrintSocketClient.processMessage(Unknown Source)
	at qz.ws.PrintSocketClient.onMessage(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.call(CallableMethod.java:70)
	at org.eclipse.jetty.websocket.common.events.annotated.OptionalSessionCallableMethod.call(OptionalSessionCallableMethod.java:68)
	at org.eclipse.jetty.websocket.common.events.JettyAnnotatedEventDriver$2.run(JettyAnnotatedEventDriver.java:210)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:748)
[TRACE] 2019-04-09 19:34:21,580 @ qz.utils.PrintingUtilities:?
	Returning processor back to pool
```

**Comment by bberenz** [comment]
It would [appear](https://github.com/qzind/tray/blob/2.0/src/qz/printer/action/PrintPixel.java#L71) that image/pdf prints don't apply a paper size if both dimensions aren't provided, so we'll just do the same for html.
Implemented in f010cb2a1e180a13e184d50f05ef194d49c86bbe

**Comment by klabarge** [comment]
Tested, works 👍
